test/lib/services/LocalCatalog.test.ts

```diff
@@ -23,7 +24,10 @@ describe('LocalCatalog', () => {
 		Sinon.restore();
 	});
 
-	describe('getRuleGroupsMatchingFilters', () => {
+	describe('getRuleGroupsMatchingFilters', async () => {
+
+		const ENGINES = await Controller.getAllEngines();
```

Comment:
Trying to remember better, but I somehow think we may have a test override for Controller. Since this is a read-only operation, we should be fine here I guess.